### ISSUE TITLE
API Suggestion

### ISSUE THREAD

**Issue by caroso1222** [bharatramnani94/ngx-notyf#1]
Hey, great initiative!

Apart from the component, I'd love to have a service exposed in the public API. I envision something along the lines of:

```typescript
import { NgxNotyf } from 'ngx-notyf';

constructor(private notyf: NgxNotyf) {}

foo() {
  this.notyf.success('Profile updated', { dismissible: true });
}
```

I know it's early days yet, just brainstorming on how I'd use this library myself.

Great work, keep it up!

**Comment by bharatramnani94** [comment]
Of course @caroso1222 , I plan to do the same.
Feel free to suggest other things as well, as I believe you are in a much better position to guide this thing.

**Comment by bharatramnani94** [comment]
The basic implementation is done and is available starting with version 0.0.2.

**Comment by caroso1222** [comment]
Good work. Let me know when you have a stable v1.x.x and I'll put it in the Notyf recipes.
